src/codecatalyst/explorer.ts

```diff
@@ -33,7 +33,11 @@ const reauth = Commands.register(
 
 async function getLocalCommands(auth: CodeCatalystAuthenticationProvider) {
     const docsUrl = isCloud9() ? codecatalyst.docs.cloud9.overview : codecatalyst.docs.vscode.overview
-    if (!isBuilderIdConnection(auth.activeConnection) || !(await auth.isConnectionOnboarded(auth.activeConnection))) {
+    if (
+        !auth.activeConnection ||
+        !auth.isConnectionValid() ||
```

Comment:
It's mildly redundant, but `auth.activeConnection` is nullable, so passing it into some subsequent calls would require non-null assertions otherwise. I'm happy to go either way here, but the redundant check felt less risky to break underneath in the case of future changes.